crates/bevy_ecs/src/schedule/system_container.rs

```diff
@@ -79,6 +80,12 @@ impl SystemContainer {
     }
 }
 
+impl Debug for SystemContainer {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "\n{{{:?}}}", &self.system())
```

Comment:
Why does this open with a newline?

Reply:
It made it easier to read a long list of systems, but I can certainly remove the newline, as typically they aren't included in Debug output.

Reply:
Yeah, I would omit this from the impl itself, and then insert it again wherever we're listing long groups of systems :)

Reply:
These fields in SystemStage make the most sense to do it for:
```rust
    /// Topologically sorted exclusive systems that want to be run at the start of the stage.
    pub(super) exclusive_at_start: Vec<SystemContainer>,
    /// Topologically sorted exclusive systems that want to be run after parallel systems but
    /// before the application of their command buffers.
    pub(super) exclusive_before_commands: Vec<SystemContainer>,
    /// Topologically sorted exclusive systems that want to be run at the end of the stage.
    pub(super) exclusive_at_end: Vec<SystemContainer>,
    /// Topologically sorted parallel systems.
    pub(super) parallel: Vec<SystemContainer>,
```
Right now I'm just doing a `#[derive(Debug)]` for `SystemStage` (it has a lot of fields).  But maybe for each of these a specialized `Debug` can be done in which if `vec.len() > 1`, we list the `SystemContainer`s each on a separate line.

Reply:
Yep, agreed!

Reply:
Implemented.